prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-enrolments/enrollee-enrolments.component.ts

```diff
@@ -12,6 +12,7 @@ import { EnrolmentStatus } from '@shared/enums/enrolment-status.enum';
 
 import { AdjudicationRoutes } from '@adjudication/adjudication.routes';
 import { AdjudicationResource } from '@adjudication/shared/services/adjudication-resource.service';
+import { EnrolmentCard } from '@shared/models/enrolment-card.model';
```

Comment:
Probably want to group shared imports for ease of scanning and future lazy loading updates